.travis.yml

```diff
@@ -11,7 +11,12 @@ go:
 install:
     # Get tools used in Makefile
     - make get-tools
-    - sudo apt-get install mtools liblzma-dev
+
+addons:
```

Comment:
Moved to using the Travis addons functionality in order to seperate the dependency management. It is this, or doing it in the makefile. I think this is cleaner.